src/Controller/UI/DatasetReviewController.php

```diff
@@ -177,31 +165,28 @@ protected function eligibiltyForReview(string $udi, Request $request)
         } else {
             $this->addToFlashDisplayQueue($request, $udi, 'notFound');
         }
+
         return $this->makeSubmissionForm($udi, $dataset, $datasetSubmission);
     }
 
     /**
      * Gets Latest dataset submission and checks for errors to add in the flash bag.
-     *
-     * @param Request $request The Symfony request object.
-     * @param Dataset $dataset A dataset instance..
-     *
-     * @return DatasetSubmission  A dataset submission instance.
+
      */
     private function latestDatasetSubmissionForReview(Request $request, Dataset $dataset)
```

Comment:
missing return type

Reply:
Yes, but decided not to do these in this go around.